core/app/models/spree/product.rb

```diff
@@ -323,7 +323,11 @@ def taxons_for_store(store)
     end
 
     def any_variant_in_stock_or_backorderable?
-      variants.any? && variants_including_master.in_stock_or_backorderable.exists?
+      variants.any?
```

Comment:
```suggestion
      if variants.any?
```

Reply:
I thought we weren't taking into consideration the master variant for the in stock or backorderable checking, as in the [purchasable?](https://github.com/spree/spree/blob/8809bed09a7a6cd5ed61e2534a62f2e98219a5f7/core/spec/models/spree/product_spec.rb#L22) case @damianlegawiec